examples/chart/teleport-cluster/templates/podmonitor.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "teleport-cluster.labels" . | nindent 4 }}
-  {{- with .Values.podMonitor.additionalLabels }}
```

Comment:
This would introduce a breaking change for those that currently use this as `additionalLabels` ?